site/tailwind.config.js

```diff
@@ -7,43 +7,32 @@ const capitalizeFirst = plugin(function ({ addUtilities }) {
       textTransform: 'uppercase'
     }
   }
-  addUtilities(newUtilities, ['responsive', 'hover'])
+  addUtilities(newUtilities)
 })
 
 module.exports = {
-  purge: ['./components/**/*.{js,ts,jsx,tsx}', './pages/**/*.{js,ts,jsx,tsx}'],
+  content: [
+    './components/**/*.{js,ts,jsx,tsx}',
+    './pages/**/*.{js,ts,jsx,tsx}'
+  ],
+  plugins: [capitalizeFirst],
   theme: {
     extend: {
-      spacing: {
-        5.5: '1.375rem',
-        7.5: '1.875rem',
-        15: '3.75rem',
-        19: '4.75rem',
-        54: '13.5rem',
-        63: '15.75rem',
-        100: '25rem',
-        128: '30rem',
-        150: '40rem',
-        200: '50rem'
-      },
       colors: {
-        vesper: '#4138AC',
-        orange: {
-          1: '#FF4D00',
-          50: '#FFFBF8',
-          100: '#FFF0E8',
-          200: '#FFE2D0',
-          300: '#FFD3B9',
-          400: '#FFC4A1',
-          500: '#FFB58A',
-          600: '#FFA773',
-          700: '#FF985B',
-          800: '#FF7D33',
-          900: '#FF7B2C',
-          950: '#FF6C15',
-          hemi: '#FF5F00',
-          opacity: '#FF6C1510'
+        grayscale: {
+          50: '#FAFAFA',
+          100: '#E5E6E6',
+          200: '#CCCCCC',
+          300: '#D4D4D8',
+          400: '#A3A3A3',
+          500: '#737373',
+          600: '#666767',
+          700: '#4D4E4E',
+          800: '#333535',
+          900: '#1A1B1B',
+          950: '#0A0A0A'
         },
+        hemi: '#FF5F00',
```

Comment:
is this reference to Hemi needed? I think everything is grey now, instead of orange.

Reply:
I left that one in purpose, just in case we want to add a link back to Hemi or something with Hemi colors. But no, it is not needed or used at the moment.